SanUI/Modules/AddonMods/coolline.lua

```diff
@@ -34,11 +34,11 @@ function S.modCoolLine(event)
 			CoolLine:ClearAllPoints()
 			if TukuiStanceBar and NumForms > 0 then 
 				CoolLine:SetPoint("BOTTOMLEFT",TukuiStanceBar,"BOTTOMRIGHT",spacing,spacing)
-				CoolLine:SetPoint("TOPLEFT",TukuiStanceBar,"TOPRIGHT",spacing,-spacing)
 			else
+				CoolLine:SetPoint("BOTTOMLEFT",TukuiActionBar1,"TOPLEFT",0,spacing)
 				CoolLine:SetPoint("TOPRIGHT",TukuiActionBar3,"TOPLEFT",-spacing,-spacing)
 			end
-			CoolLine:SetPoint("BOTTOMRIGHT",TukuiActionBar3,"BOTTOMLEFT",-spacing,spacing)
+			CoolLine:SetPoint("BOTTOMRIGHT",TukuiActionBar3,"BOTTOMLEFT",-spacing,-spacing)
```

Comment:
What's the reason for this change?

Reply:
pixel perfect symmetrical alignment. Spacing above and below of coolline. See BOTTOMRIGHT anchor...

Reply:
Oh I get it, but it was intentional this way, I wanted it aligned with the buttons themselves, not the bar backgrounds (which is what `TukuiActionBar3` is referring to). I'd prefer if you did not change this.

And if you look closely, in the screenshot without stance bar, it does look off in exactly this way.

Reply:
If this is how you want it,
![image](https://user-images.githubusercontent.com/457645/211151947-5684fe39-9298-4602-8a01-84f8526857de.png)

then the code should be:
```lua
CoolLine:ClearAllPoints()
if TukuiStanceBar and NumForms > 0 then 
	CoolLine:SetPoint("TOPLEFT",TukuiStanceBar,"TOPRIGHT",spacing,-spacing)
	CoolLine:SetPoint("BOTTOMRIGHT",TukuiActionBar3,"BOTTOMLEFT",-spacing,spacing)
else
	CoolLine:SetPoint("LEFT",TukuiActionBar1,"LEFT")
	CoolLine:SetPoint("BOTTOM",TukuiActionBar3,"BOTTOM", 0, spacing)
	CoolLine:SetPoint("TOPRIGHT",TukuiActionBar3,"TOPLEFT",-spacing,-spacing)
end
S.placeStanceBar()
```